src/StepperConfiguration.hpp

```diff
@@ -40,8 +40,8 @@ struct Ra {
     using stepper_slew = Stepper<interrupt, driver, ramp_slew>;
     using stepper_trk  = Stepper<interrupt, driver, ramp_trk>;
 
-    constexpr static float SPEED_COMPENSATION
-        = static_cast<float>(interrupt::FREQ) / static_cast<float>(ramp_slew::interval(1)) - (SPEED_TRK * 2.0f);
+    constexpr static float SPEED_COMPENSATION = SPEED_SLEW;
+    // = static_cast<float>(interrupt::FREQ) / static_cast<float>(ramp_slew::interval(1)) - (SPEED_TRK * 2.0f);
```

Comment:
I don't understand why you're removing this?

Reply:
Because it doesn't work. It causes long delays after slews because it is way too low. And Andre has no time to investigate why or explain why this calculation is done the way it is.

Reply:
Gotcha, remove the commented out code then